### PR TITLE
fix: Fix wrong event type in onHover

### PR DESCRIPTION
**Pull request by nilscognite**
Change from PointEvent to MouseEvent everywhere when hover

And update to "version": "4.21.1

### CODE REVIEW

File: viewer/packages/utilities/src/events/PointerEvents.ts [hunk 1]
```diff
@@ -7,7 +7,7 @@
  * @beta
  */
 export class PointerEvents {
-  public onHover(_event: PointerEvent): void {}
```
Comment (pramodcog) [inline viewer/packages/utilities/src/events/PointerEvents.ts:10]: Why Hover event is changed to `MouseEvent`?
`PointerEvent` basically inherits `MouseEvent`, if `MouseEvent` is used than Touch/Pen will not work

Comment (nilscognite) [reply viewer/packages/utilities/src/events/PointerEvents.ts:10]: I know. But hover is never used on Touch. The position of your finders are unknown anyway. So I wanted to synchronize it. sometimes it uses Mouse and sometime Pointer.

Comment (nilscognite) [reply viewer/packages/utilities/src/events/PointerEvents.ts:10]: I will update the API doc when you have approved.

**Review by pramodcog — APPROVED** [review]
LGTM!
Can you make sure API docs are updated as well

**Pull request merged**